x-pack/plugins/observability_solution/infra/public/components/asset_details/hooks/use_custom_dashboards_permissions.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import { useKibanaContextForPlugin } from '../../../hooks/use_kibana';
+
+export const useCanManipulateCustomDashboard = () => {
```

Comment:
nit:
```suggestion
export const useCustomDashboardUserPermissions = () => {
```

Reply:
@kpatticha , following up on your comment, perhaps the name here could be `useSavedObjectUserPermissions`?

Reply:
Renamed, thanks 👍